### PR TITLE
repo documentation

### PR DESCRIPTION
**Pull request by konnofuente**
Here i updated the project
-contribution
-readme and 
-issue template

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+
+Inspired by Sparkbox's awesome article on semantic commit messages. Please use the following commit message format:
+
+- **chore** (updating npm tasks etc; no production code change) -> `git commit -m 'chore: commit-message-here'`
+- **docs** (changes to documentation) -> `git commit -m 'docs: commit-message-here'`
+- **feat** (new feature) -> `git commit -m 'feat: commit-message-here'`
+- **fix** (bug fix) -> `git commit -m 'fix: commit-message-here'`
+- **refactor** (refactoring production code) -> `git commit -m 'refactor: commit-message-here'`
+- **style** (formatting, missing semi colons, etc; no code change) -> `git commit -m 'style: commit-message-here'`
+- **test** (adding missing tests, refactoring tests; no production code change) -> `git commit -m 'test: commit-message-here'`
```
Comment (dilane3) [inline .github/ISSUE_TEMPLATE/CONTRIBUTING.md:227]: Excellent 👏🏼

File: README.md [hunk 2]
```diff
@@ -1,2 +1,117 @@
...
+- [Contributing](#contributing)
+- [Present Contributors](#present-contributors)
+- [Resources](#resources)
+- [Communication](#communication)
+- [Code of Conduct](#code-of-conduct)
+- [License](#license)
+
+## Introduction
+
+Rasengan.js is a powerful framework for building modern web applications with ease. This repository provides pre-prepared templates that you can use to kickstart your projects. Whether you are building a chatbot, an e-commerce site, a portfolio, or a dashboard, we have a template for you.
```
Comment (dilane3) [inline README.md:29]: I like this intro 👏🏼

File: README.md [hunk 3]
```diff
@@ -1,2 +1,117 @@
...
+
+3. Create a new project using a template:
+   ```bash
+   rasengan create <template-key>
+   ```
+
+   For example, to create a project using the Gemini chatbot template:
+   ```bash
+   rasengan create chatbot/gemini
+   ```
```
Comment (dilane3) [inline README.md:72]: This section has to be updated, it's not the right way to use templates in Rasengan.Js

What developers will do is:

- Explore the templates from a website **ex: templates.rasengan.dev**
- Identify the template that the developer want by copying its **template-key**
- Use the Rasengan CLI (without inslalling it explicitly) and generate a new project based on the template he has chosen. **ex: npx create-rasengan@latest [project-name] --template <template-key>** 

The **[project-name]** is not required in this command, so if the developer execute it without mentioning the [project-name] the CLI will ask him the name of the project later during the execution process.

Typically, that's the process.

File: .github/ISSUE_TEMPLATE/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+│   ├── atoms/
+│   │   ├── Icon/
+│   │   ├── Label/
+│   ├── molecules/
+│   │   ├── Navbar/
+│   │   ├── Footer/
+│   ├── organisms/
+│   │   ├── DashboardPage/
+│   │   ├── ProfilePage/
+```
```
Comment (dilane3) [inline .github/ISSUE_TEMPLATE/CONTRIBUTING.md:98]: We have to discuss again for this section.

File: .github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+Before opening an issue:
+
+- [Search for duplicate or closed issues](https://github.com/coreui/coreui-free-bootstrap-admin-template/issues?utf8=%E2%9C%93&q=is%3Aissue)
+- Prepare a [reduced test case](https://css-tricks.com/reduced-test-cases/) for any bugs
+- Read the [contributing guidelines](https://github.com/coreui/coreui-free-bootstrap-admin-template/blob/main/.github/CONTRIBUTING.md)
```
[the remark this answers is not in the document]
Comment (konnofuente) [reply .github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.md:5]: Ohh yes that true we need to clear that

File: .github/ISSUE_TEMPLATE/CONTRIBUTING.md (hunk 4 above)
Comment (konnofuente) [reply .github/ISSUE_TEMPLATE/CONTRIBUTING.md:98]: okay

File: .github/ISSUE_TEMPLATE/CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,240 @@
...
+- Define clear types and interfaces.
+- Use `strict` mode in `tsconfig.json`.
+- Avoid using `any` type.
+
+#### Tailwind CSS
+
+- Use utility-first approach.
+- Define custom themes in `tailwind.config.js`.
+- Avoid using inline styles when Tailwind classes suffice.
+
```
[the remark this answers is not in the document]
Comment (konnofuente) [reply .github/ISSUE_TEMPLATE/CONTRIBUTING.md:126]: resolve

**Review by dilane3 — APPROVED** [review]

**Pull request merged**